### PR TITLE
Issue 8038: Scheduled posts via the frontend

### PR DESCRIPTION
**Pull request by annando**
This PR partially fulfills the request in issue #8038. We can now define a schedule date for a newly created post in the frontend. This triggers the same functionality that is used for scheduled post via the API.

The current status is that one can only see and delete these scheduled posts via the API.

This PR is more or less experimental. It shouldn't break existing functionality but I won't guarantee that it works flawlessly in all circumstances.

### CODE REVIEW

**Comment by tobiasd** [comment]
Can the same date input be included in the share block for composing events?

**Comment by annando** [comment]
@tobiasd that wouldn't be that easy.

File: view/templates/item/compose.tpl [hunk 1]
```diff
@@ -81,6 +81,8 @@
 			<div class="jotplugins">
 				{{$jotplugins nofilter}}
 			</div>
+
+			{{if $scheduled_at}}{{$scheduled_at nofilter}}{{/if}}
```
Comment (MrPetovan) [inline view/templates/item/compose.tpl:85]: Is it an optional field?

Comment (annando) [reply view/templates/item/compose.tpl:85]: I had been unsure if possibly this could be something that could be made optional in the end.

**Comment by tobiasd** [comment]
> @tobiasd that wouldn't be that easy.

Ok, and it is not that important as events are put in the calendar and shown there anyhow. Though I can see some people wanting their events posted at a specific time for the same reason as some want to have their regular postings posted at a specific time.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**